### PR TITLE
Reformatting files

### PR DESCRIPTION
**Pull request by OliverOverend**
I have made Pong Game into a Python script and reformatted the two files using a tool called Black.

### CODE REVIEW

File: Nelson's Trivia.py [hunk 1]
```diff
@@ -1,36 +1,39 @@
-print('hello, welcome to nelson trivia!')
-ans = input('are you ready to play (yes/no):')
+print("hello, welcome to nelson trivia!")
+
```
Comment (OliverOverend) [inline Nelson's Trivia.py:2]: I have added some white space here for readability.

File: Pong Game.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 wn.tracer(0)
 
 
-#Paddle A
+# Paddle A
```
Comment (OliverOverend) [inline Pong Game.py:11]: There should be a single space after the hash.

File: Nelson's Trivia.py [hunk 3]
```diff
@@ -1,36 +1,39 @@
...
-    
-ans = input('1. What is my favourite japanese dish?')
-if ans.lower() == 'katsu curry':
-        score +=1
-        print('correct')
+    print("you suck")
+
+ans = input("1. What is my favourite japanese dish?")
+if ans.lower() == "katsu curry":
+    score += 1
```
Comment (OliverOverend) [inline Nelson's Trivia.py:15]: There should be a single space after the =.

**Review by NelsonCHChan — APPROVED** [review]

**Comment by OliverOverend** [comment]
Pls can you merge the changes?

**Comment by NelsonCHChan** [comment]
I’m baffed thought I did😂

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by OliverOverend** [comment]
Is there a green button that says merge changes?

**Comment by NelsonCHChan** [comment]
think i did it!

**Pull request merged**